gslib/tests/test_mb.py

```diff
@@ -123,7 +131,11 @@ def test_create_with_retention_invalid_arg(self):
                              suri(bucket_uri)],
                             expected_status=1,
                             return_stderr=True)
-    self.assertRegexpMatches(stderr, r'Incorrect retention period specified')
+    if self._use_gcloud_storage:
+      self.assertIn("Duration unit 'econd' must be preceded by a number",
```

Comment:
second?

Reply:
The "s" is actually a valid unit, so it gets cut. Will add a comment.